### ISSUE TITLE
Message sent, Receiver got an error

### ISSUE THREAD

**Issue by cstiens** [zom/Zom-iOS#445]
T sent Ro messages. T gets double checks. Ro receives the message, but it shows an error on the message.

![bc2d1234-28b7-48a2-8e03-f83b19613071](https://user-images.githubusercontent.com/1434597/30979701-0a23af42-a444-11e7-9de9-49769fbaf54e.jpg)

**Comment by n8fr8** [comment]
That isn't an error, its a feature. It is saying Tiffrobo is sending from a new device, or with a new key, and you need to go to the profile to approve the new key.

What happens when you click "View Profile"?

**Comment by cstiens** [comment]
Right. I think we need to address the UI once we improve the verification workflow, because it looks like an error. The other part of this, was that Ro wasn't able to send T messages. I think because she hadn't verified her. This may be intended right now... but it's very confusing for the end user because there's no explanation.

> What happens when you click "View Profile"?

If you open the profile, there's a list of 4 keys for Tiffrobo and 2 separate buttons to 'Verify Contact'.
